### PR TITLE
Remove external sorters test from code coverage tests

### LINKED ISSUE #1392: Singularity tests failing on master
Full tests are failing on master:
https://github.com/SpikeInterface/spikeinterface/actions/runs/4323218924/jobs/7546566323

Related to #1386
I think the reason isthat the new singularity tests don't have access to the dev path. We can do two things:
* Add the dev path to the full tests so the singularity tests are ran there.
* Remove the singularity tests from the full tests with marking machinery.

What's better?

### PR DESCRIPTION
**Pull request by h-mayorquin**
The solution that @alejoe91  requested for #1392.

Should close #1392 when completed.

### CODE REVIEW

File: .github/workflows/full-test-with-codecov.yml [hunk 1]
```diff
@@ -86,7 +88,7 @@ jobs:
       - name: run tests
         run: |
           source ~/test_env/bin/activate
-          pytest --cov=./ --cov-report xml:./coverage.xml -vv -ra --durations=0 | tee report_full.txt; test ${PIPESTATUS[0]} -eq 0 || exit 1
+          pytest -m "not sorters_external" --cov=./ --cov-report xml:./coverage.xml -vv -ra --durations=0 | tee report_full.txt; test ${PIPESTATUS[0]} -eq 0 || exit 1
```
Comment (alejoe91) [inline .github/workflows/full-test-with-codecov.yml:91]: smart :)

**Comment by alejoe91** [comment]
Thanks @h-mayorquin 

I guess we wait for tests to pass, then you're going to remove the `on: pull_request`?

**Comment by h-mayorquin** [comment]
@alejoe91 Yep, now they are completed without errors. This should be ready to go.

**Review by alejoe91 — APPROVED** [review]
Awesome!!!

**Pull request merged**